### DISCUSSION TITLE
Which actions affects the reputation?

### DISCUSSION THREAD

**Discussion by Kiskadee-dev** [Vdauphin/HeartsAndMinds#1416] [Question and answer] [answered]
Hi, I'm looking to see what might be affecting our group's reputation as shortly after the server starts with the rep as normal it changes to low and after a while to very low even while completing many missions, giving bananas, healing the crazy drivers and cleaning the paintings on the streets

Does AI driving around damaging walls and fences affect it?

**Comment by Vdauphin** [comment, marked as answer]
Hello,
See https://github.com/Vdauphin/HeartsAndMinds/discussions/1408
Cheers

**Reply by Kiskadee-dev** [reply]
Hello!

I'm testing it, looks like CUP walls doesn't behave well with isKindOf

Map: Kujari
![image](https://user-images.githubusercontent.com/10299707/193623813-bb161275-34ec-409e-af5b-09f35b1c1d77.png)
![image](https://user-images.githubusercontent.com/10299707/193623872-a9bff4bd-c1c0-4c90-b0c8-7ca99c214727.png)


On Altis it works fine

![image](https://user-images.githubusercontent.com/10299707/193625107-9e765699-f7f5-49fe-9973-270499d1a89f.png)
![image](https://user-images.githubusercontent.com/10299707/193625240-d7dca4be-b2aa-426d-b24e-10a7e21738e0.png)

**Reply by Vdauphin** [reply]
Looks like wall have `House` as parent
![Capture d’écran du 2022-10-03 18-49-21](https://user-images.githubusercontent.com/14364400/193633824-e43d8cac-9ee7-416b-8b85-741b7374ca96.png)

**Reply by Vdauphin** [reply]
See https://github.com/Vdauphin/HeartsAndMinds/pull/1421

**Reply by Kiskadee-dev** [reply]
Testing the PR, there are more objects that has the same issue

CUP_A2_2m5_corner_ep1
![image](https://user-images.githubusercontent.com/10299707/193648412-043a8a98-ccbe-42f5-a53e-c949035a8c1c.png)

CUP_A1_Plot_Ohrada
![image](https://user-images.githubusercontent.com/10299707/193649514-08671bc4-cd14-4d86-86f4-1426099acb21.png)

CUP_A1_Plot_Ohrada_Zlomena
![image](https://user-images.githubusercontent.com/10299707/193649725-2af98cbe-f538-44ff-8cca-9160d7a220a0.png)

And probably other variations of walls, but i think the corners do have 'corner' in the name and could be removed aswell

The last 2 are okish as they were far from the roads but i'm not sure if other maps uses them as common fences like Altis does, other fences are ok as i think ace detects them
